src/containers/category-dropdown/CategoryDropdown.tsx

```diff
@@ -20,25 +20,29 @@ import {
   CreateCategoriesParams,
   ErrorResponse
 } from '~/types'
-import { styles } from '~/containers/category-dropdown/CategoryDropdown.styles'
 
 interface CategoryDropdownInterface {
   category: string | null
   onCategoryChange: (
     _: SyntheticEvent,
     value: CategoryNameInterface | null
   ) => void
+  authorizedCreateCategory: boolean | null
+  label: string | null
```

Comment:
you can just make it optional, no need in lable being null

Reply:
https://github.com/Project-Based-10-24-vk/Client/pull/125/commits/413a34dce34c9a713a5cd367b8cb915bc1b40133